src/features/Library/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx

```diff
@@ -0,0 +1,74 @@
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import { FormControlLabelProps } from '@mui/material';
+
+import { Modal } from 'components';
+import { RadioGroupController } from 'components/FormComponents/RadioGroupController';
+
```

Comment:
remove space pls, add RadioGroupController to according index.ts to import just from the components/FormComponents